src/ray/core_worker/transport/dependency_resolver.cc

```diff
@@ -42,7 +42,7 @@ void DoInlineObjectValue(const ObjectID &obj_id, std::shared_ptr<RayObject> valu
   RAY_CHECK(found) << "obj id " << obj_id << " not found";
 }
 
-void LocalDependencyResolver::ResolveDependencies(const TaskSpecification &task,
```

Comment:
Why no more const?

Reply:
Since this mutates the task, it is weird to call it const.